### PR TITLE
white space issue was solved and license headers were added

### PR DESCRIPTION
**Pull request by ravindraranwala**

### CODE REVIEW

File: components/mediation-ntask/src/main/java/org/wso2/carbon/mediation/ntask/NTaskTaskManager.java [hunk 1]
```diff
@@ -533,88 +548,88 @@ private String managerId() {
...
-	@Override
-	public void addObserver(TaskManagerObserver o) {
-		if (observers.contains(o)) {
-			return;
-		}
-		observers.add(o);
-	}
+    @Override
+    public void addObserver(TaskManagerObserver o) {
+        if (observers.contains(o)) {
```
Comment (shafreenAnfar) [inline components/mediation-ntask/src/main/java/org/wso2/carbon/mediation/ntask/NTaskTaskManager.java]: Can't we have meaning full name here for reference "o" ?

File: components/mediation-ntask/src/main/java/org/wso2/carbon/mediation/ntask/NTaskTaskManager.java [hunk 2]
```diff
@@ -533,88 +548,88 @@ private String managerId() {
...
-	private void updateAndCleanupObservers() {
-		Iterator<TaskManagerObserver> i = observers.iterator();
-		while (i.hasNext()) {
-			TaskManagerObserver observer = i.next();
-			observer.update();
-			i.remove();
-		}
-	}
+    private void updateAndCleanupObservers() {
+        Iterator<TaskManagerObserver> i = observers.iterator();
```
Comment (shafreenAnfar) [inline components/mediation-ntask/src/main/java/org/wso2/carbon/mediation/ntask/NTaskTaskManager.java]: Having a better name for "i" would increase the readability of your code.

File: components/mediation-ntask/src/main/java/org/wso2/carbon/mediation/ntask/NTaskTaskManager.java (hunk 1 above)
Comment (ravindraranwala) [inline components/mediation-ntask/src/main/java/org/wso2/carbon/mediation/ntask/NTaskTaskManager.java]: This was fixed in pull request https://github.com/wso2/carbon-mediation/pull/258

File: components/mediation-ntask/src/main/java/org/wso2/carbon/mediation/ntask/NTaskTaskManager.java (hunk 2 above)
Comment (ravindraranwala) [inline components/mediation-ntask/src/main/java/org/wso2/carbon/mediation/ntask/NTaskTaskManager.java]: This was fixed in pull request https://github.com/wso2/carbon-mediation/pull/258

**Pull request merged**